x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/__mocks__/request_responses.ts

```diff
@@ -109,7 +109,7 @@ export const getCreateRequest = (): ServerInjectOptions => ({
 export const createActionResult = (): ActionResult => ({
   id: 'result-1',
   actionTypeId: 'action-id-1',
-  description: '',
+  name: '',
```

Comment:
@FrankHassanabad send-ahead that we're changing the action property `description` to `name`.  Currently it looks like actions aren't really used by the siem code ATM, but do have some references in these mocks.